cdk/lib/payment-api.ts

```diff
@@ -322,6 +322,11 @@ export class PaymentApi extends GuStack {
       snsTopicName: `alarms-handler-topic-${this.stage}`,
     });
 
+    const stripeExpressMetricDuration = Duration.minutes(5);
+    const stripeExpressEvaluationPeriods = 18; // The number of 5 minute periods in 90 minutes
+    const stripeExpressAlarmPeriod = Duration.minutes(
+      stripeExpressMetricDuration.toMinutes() * stripeExpressEvaluationPeriods
+    );
```

Comment:
just food for though, but i tend to prefer grouping these values under an object. Are there any preferences on this independent variables vs grouped ones? 
```suggestion
        const stripeExpressMetric = {
        duration: Duration.minutes(5);
        evaluationPeriods: 18; // The number of 5 minute periods in 90 minutes
        alarmPeriod = Duration.minutes(stripeExpressMetricDuration.toMinutes() * stripeExpressEvaluationPeriods
     }
```